config.json

```diff
@@ -2224,8 +2224,15 @@
         "slug": "darts",
         "name": "Darts",
         "uuid": "6c64649b-ea81-4118-9e74-a0a55018ffbc",
-        "practices": [],
-        "prerequisites": [],
+        "practices": [
+          "math"
+        ],
```

Comment:
The `javascript` track currently leaves this empty purposefully.

Reply:
Thanks! I removed it 🙏